core/app/models/spree/promotion/rules/country.rb

```diff
@@ -0,0 +1,23 @@
+# A rule to limit a promotion based on shipment country.
+module Spree
+  class Promotion
+    module Rules
+      class Country < PromotionRule
+        preference :country_id, :integer
+
+        def applicable?(promotable)
+          promotable.is_a?(Spree::Order)
+        end
+
+        def eligible?(order, options = {})
+          country_id = options[:country_id] || order.ship_address.try(:country_id)
```

Comment:
@Mafi88 we can add a delegation for country_id through ship_address in order model.